modules/storage-multi-attach-error.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-storage-issues.adoc
+
+[id="storage-multi-attach-error_{context}"]
+= Resolving multi-attach error issues
+
+.Problem
+When a node crashes or shuts down abruptly, the ReadWriteOnce (RWO) volume is expected to be unmounted from the node so that it can be used by a Pod scheduled on another node. However, this is not possible because when the node is down, the system cannot unmount the volume. A multi-attach error message is reported, for example:
+
+[source,terminal]
+.Example output
+--
+Unable to attach or mount volumes: unmounted volumes=[sso-mysql-pvol], unattached volumes=[sso-mysql-pvol default-token-x4rzc]: timed out waiting for the condition
+Multi-Attach error for volume "pvc-8837384d-69d7-40b2-b2e6-5df86943eef9" Volume is already used by pod(s) sso-mysql-1-ns6b4
+--
+
+.Solution
+.. *Use RWX volumes.*
++
+For most storage solutions, the solution to the multi-attach error is to use ReadWriteMany (RWX) volumes.
++
+.. *Recover or delete the failed node (vSphere).*
++
+VMware vSphere volumes do not support RWX, so RWO volumes must be used instead. However, RWO volumes cannot be mounted on multiple nodes. 
++
+If a node fails, the system does not allow the attached RWO volume to be mounted on a new node because it is already assigned to the failed node.
```

Comment:
I think this sentence should be in the 'Problem' section.

Reply:
Removed it here. It was already similarly stated in the Problem section, which I've reworded for more clarity.